testsuite/forge-cli/src/main.rs

```diff
@@ -1420,6 +1421,65 @@ fn realistic_env_max_throughput_test_suite(duration: Duration) -> ForgeConfig {
         )
 }
 
+fn realistic_network_tuned_for_throughput_test() -> ForgeConfig {
+    ForgeConfig::default()
+        .with_initial_validator_count(NonZeroUsize::new(12).unwrap())
+        // if we have full nodes for subset of validators, TPS drops.
+        // Validators without VFN are proposing almost empty blocks,
```

Comment:
Seems unexpected. I would think QS batch broadcast has an equal opportunity to make it to a VN-only node vs. VN+VFN node proposal. Is there a forge run you can point me to here?

Reply:
Makes sense - maybe we can change the comment to reflect this since quorum store is the default. If its already landing I'll go hunt down the other test too and change them

Reply:
I'll update in the stacked PR